python/mxnet/symbol.py

```diff
@@ -51,11 +51,31 @@ def __repr__(self):
                             'Grouped' if name is None else name)
 
     def __iter__(self):
-        """Returns all outputs in a list"""
+        """Returns a generator object of symbol.
+
+        One can loop through the returned object to get outputs.
```

Comment:
Is the object a list? If so, let's call it a list. Sounds a bit strange to say "loop through the object"